polling_stations/apps/data_finder/views.py

```diff
@@ -135,6 +136,7 @@ class PostcodeView(BasePollingStationView):
 
     def get(self, request, *args, **kwargs):
         rh = RoutingHelper(self.kwargs['postcode'])
+        self.do_not_log = request.GET.get('no-log') is not None
```

Comment:
I think this can either remove the `is not None` as if `self.do_not_log` is None it will still evaluate to `False`, or it could be `request.GET.get('no-log', False)`.